packages/shopify-api/lib/fulfillment-service/types.ts

```diff
@@ -0,0 +1,32 @@
+import {AdapterArgs} from '../../runtime/types';
+
+export interface FulfillmentServiceParams extends AdapterArgs {
+  /**
+   * The raw body of the request.
+   */
+  rawBody: string;
+}
+
+export enum FulfillmentServiceErrorReason {
```

Comment:
I think this type ended up being very similar to the flow one, right? Should we consider creating a shared utils for validating the HMAC + body combo with its own types, and then having these extend from those?

I think this could be used for:
- flow
- fulfillment services
- webhooks

more to come?